src/processors/mod.rs

```diff
@@ -1,3 +1,4 @@
 //! Default processors.
 
 pub mod rendering;
+pub mod transform;
```

Comment:
Odd, my text editor says there are new lines in my file, I'll try re-pushing that.